packages/smart-contracts/scripts-create2/contract-setup/setupErc20ConversionProxy.ts

```diff
@@ -18,14 +20,13 @@ export const setupErc20ConversionProxy = async (
   // Setup contract parameters
   const Erc20ConversionProxyContract = new hre.ethers.Contract(
     contractAddress,
-    erc20ConversionProxy.getContractAbi(),
+    erc20ConversionProxy.getContractAbi(ERC20ConversionVersion),
```

Comment:
Why are we not using the latest version? Just curious

Reply:
Within erc20Conversion's index, most of the addresses are registered within version "0.0.1", but their real ABI is "0.1.2".
This contract indexes is quite confusing. If someone want to update its setUp, he may have to choose the version...